test/cluster/base_environment.py

```diff
@@ -398,7 +398,10 @@ def get_tablet_ip_port(self, tablet_name):
     """
     tablet_info = json.loads(self.vtctl_helper.execute_vtctl_command(
         ['GetTablet', tablet_name]))
-    return '%s:%s' % (tablet_info['ip'], tablet_info['port_map']['vt'])
+    host = tablet_info['hostname']
+    if ':' in host:
```

Comment:
A comment would have been good here to explain why you're doing this.

It may not be obvious to other people that this meant to address IPv6 addresses.